core-schema/src/main/resources/sql-schema/V8__Mod_clans.sql

```diff
@@ -1,10 +1,11 @@
 CREATE TABLE clans_clan_info (
   clan_id INT AUTO_INCREMENT PRIMARY KEY,
-  clan_name VARCHAR(32) NOT NULL,
+  clan_name VARCHAR(32) UNIQUE NOT NULL,
```

Comment:
This is a step in the right direction, but the code needs to be updated to account for this change.

1. The `createClan` method javadocs need to be updated to note what happens when the clan by the attempted name already exists. The implementation likely needs to be updated too.
2. Wasn't the reason we wanted a unique clan name so that we can select by the clan name? I think you wanted to add a method `Optional<Clan> getClanByName(Transaction transaction, String name)`.

Reply:
I completely forgot! will do.